CONTRIBUTING.md

```diff
@@ -2,57 +2,99 @@
-
-Once the topic branch exists, pull requests can be made against it in the usual way. It may also be brought up to date with new changes merged into master by anyone with commit access, if the changes produce merely a fast-forward merge. However, if changes from the master branch create a new merge commit, that commit needs to be reviewed in a pull request.
-
-Changes made in a topic branch can be merged into master by creating and then [resolving in the normal way](#issue_resolution) a pull request against the master branch.
+When you submit or change your pull request, the Travis build system will 
+automatically run tests to ensure valid schema syntax. If your pull request 
+fails to pass tests, review the test log, make changes and then push them to 
+your feature branch to be tested again.
 
 <a name="issue_resolution"></a>
 ## Issue Resolution
 
-Once a pull request or issue have been submitted, anyone can comment or vote on an issue to express their opinion following the Apache voting system. Quick summary:
+Once a pull request or issue have been submitted, anyone can comment or vote on 
+to express their opinion following the Apache voting system. Quick summary:
 
 - **+1** something you agree with
-- **-1** if you have a strong objection to an issue, which will be taken very seriously. A -1 vote should provide an alternative solution.
+- **-1** if you have a strong objection to an issue, which will be taken very 
+seriously. A -1 vote should provide an alternative solution.
 - **+0** or **-0** for neutral comments or weak opinions.
 - It's okay to have input without voting
 - Silence gives assent
 
-A pull request with at least two **+1** votes, no **-1** votes, and that has been open for at least 3 days, is ready to be merged. The merge should be done by someone from a different organization than the submitter. (We sometimes waive the 3 days for cosmetic-only changes -- use good judgment.)
-
-If an issue gets any **-1** votes, the comments on the issue need to reach consensus before the issue can be resolved one way or the other. There isn't any strict time limit on a contentious issue.
-
-The project will strive for full consensus on everything until it runs into a problem with that model.
+A pull request with at least two **+1** votes, no **-1** votes, that has been 
+open for at least 3 days, and whose cross-referenced pull requests to server 
+and compliance have similarly been upvoted is ready to be merged. The merge 
+should be done by someone from a different organization than the submitter. 
+(We sometimes waive the 3 days for cosmetic-only changes -- use good judgment.) 
+A pull request to either the schemas, servers or compliance repository that 
+involves changes to the others should not be merged without coordinating, 
+mergable pull requests to the other repositories. Conversely, when merging a 
+pull request the other pull requests in the branch set must be merged at the 
```

Comment:
Both the reference server and compliance suite depend to schemas,
but not each other.

All need to be approved and ready to merge before any are
merged.  Merging the schemas first will be necessary so that
the CI tests for server/compliance complete when they are merged.

CH Albach <notifications@github.com> writes:
> Tactical question:
> I can't tell whether the reference server or compliance suites are actually
> depending on the schemas repo directly. If they are, does it stand to reason
> that you would need to merge the schemas PR ahead of the other two?